docs/upgrade/index.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: documentation
+title: Katello Upgrade
+sidebar: sidebars/documentation.html
+---
+
+# Katello Upgrade
+
+Katello supports upgrades from version 2.0.  For users transitioning from 1.4, please see - [Transition Guide](/docs/{{ site.version }}/installation/2.0-transition.html).
+
+## Step 1 - Backup
+
+If Katello is running on a Virtual Machine, it would be a good idea to take a snapshot prior to upgrading. Otherwise, take a backup of the relevant databases by following the instructions here (TODO).
```

Comment:
Here is a link to our older backup instructions you could start from - https://fedorahosted.org/katello/wiki/GuideServerBackups  they were on the list of items to get ported over and updated on the Redmine wiki. There may even be an open issue for tracking it.